content/blog/2022/2022-04-08-interactions.md

```diff
@@ -0,0 +1,37 @@
++++
+author = "Noah Novšak"
+date = "2022-04-08"
+draft = true
+title = "Predictive Modelling with Attribute Interactions"
+type = "blog"
+thumbImage = "/blog_img/2022/image-20220408145950556.png"
+frontPageImage = "/blog_img/2022/image-20220408145950556.png"
+blog = ["orange"]
+shortExcerpt = "Interactions widget in Orange 3"
+longExcerpt = "The Interactions widget has been added back to Orange 3. Illustrating how to use attribute interactions to improve predictive models."
+x2images = true  # true if using retina screenshots, else false
++++
+
+The *Interactions* widget is one of the newest additions to Orange. Previously only available in Orange 2, it has been rewritten and is accessible as a prototype add-on. This way, one need not go through the trouble of [compiling older versions](/blog/2022/2022-01-10-orange2/) anymore.
+
+But what does it do?
+
+It computes and displays the [interaction](http://stat.columbia.edu/~jakulin/Int/) between attributes by calculating the mutual information between them and a third target variable. Doing so provides insight into the data at hand and aids in the search for better visualizations and predictive models.
+
+As far as visualizations go, consider [Scatter Plot](/widget-catalog/visualize/scatterplot/). Connecting Interactions to the *Features* input allows the user to manually select the projection, similarly to the *find informative projections* button, but with a little more oomph. Enabling sorting, filtering, attribute selection, and offering a clear view into, where the information is coming from, be it from one single feature or a specific combination.
+
+Another, possibly even more intriguing application would be, as mentioned, improving the performance of prediction models. To illustrate, let's take a look at the *MONK* dataset, readily available directly in Orange. First, we can try and see how well a Naive Bayes Classifier (NBC) can predict the target variable like so.
```

Comment:
"Readily available directly in Orange". I would add "in the Datasets widget" so the reader know where to look.